search/retrieval-augmented-generation/examples/rag_google_documentation.ipynb

```diff
@@ -85,46 +91,16 @@
    },
    "outputs": [],
    "source": [
-    "!pip install --user google-cloud-aiplatform==1.36.0"
+    "%pip install -q --upgrade --user google-cloud-aiplatform==1.36.1"
```

Comment:
Do we need to do the `--user` option in Colabs? I forget if the default Python installation is writable by the notebook user. If we don't need this, I would remove it.

Reply:
Most of the other notebooks in this repository include it, so I added it in to be consistent.